src/middlewares/jwtValidator.ts

```diff
@@ -0,0 +1,17 @@
+import jwt from 'jsonwebtoken';
+import {Request, Response, NextFunction} from 'express';
+import {JWT_SECRET_KEY} from "../constants/constants";
+
+// @ts-ignore
+export const jwtValidator = (req: Request, res: Response, next: NextFunction) => {
+  let accessToken = req.headers['access-token'] as string;
+  if (!accessToken) {
+    return res.status(401).send("Unauthorized user");
```

Comment:
Move response send to error handler middleware.
Send error to error handler middleware as Error class with next(error)

Reply:
doen

Reply:
donedone